src/osgPlugins/fbx/CMakeLists.txt

```diff
@@ -18,27 +21,22 @@ SET(TARGET_H
     ReaderWriterFBX.h
     WriterCompareTriangle.h
     WriterNodeVisitor.h
+    fbxTextureUnitMap.h
+    fbxUtil.h
+    fbxMatrixTransformVisitor.h
 )
 
-IF(CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
It is unclear that we are doing the right thing here. I think we may want both the 3.6 changes and our changes here. Maybe it should look like this:
```
INCLUDE_DIRECTORIES(${BOOST_INCLUDE_DIR})

link_directories(${BOOST_LIB_DIR})

IF(CMAKE_COMPILER_IS_GNUCXX)
SET(CMAKE_CXX_FLAGS "-w")
ELSEIF(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
    REMOVE_CXX_FLAG(-pedantic)
    REMOVE_CXX_FLAG(-Wall)
    REMOVE_CXX_FLAG(-Wunused)
    REMOVE_CXX_FLAG(-Wextra)
ENDIF()

IF(WIN32)
    OPTION(FBX_SHARED OFF)
    IF(FBX_SHARED)
        ADD_DEFINITIONS(-DFBXSDK_SHARED)
    ENDIF(FBX_SHARED)

    SET(TARGET_EXTERNAL_LIBRARIES wininet)
ENDIF(WIN32)

```